support/k8s/k8s-workload-registrar/controller.go

```diff
@@ -153,7 +152,7 @@ func (c *Controller) createPodEntry(ctx context.Context, pod *corev1.Pod) error
 }
 
 func (c *Controller) getFederationDomains(pod *corev1.Pod) []string {
-	if val, ok := pod.GetAnnotations()[c.c.FederationAnnotation]; ok {
+	if val, ok := pod.GetAnnotations()["spiffe.federatesWith"]; ok {
```

Comment:
The previous default in the PR was `spiffe.io/federatesWith`? Why the change to `spiffe.federatesWith`?

Also, this value should be a constant somewhere.

Reply:
I moved it all to its own package